spec/requests/api/v1/docs/reactions_spec.rb

```diff
@@ -66,6 +66,54 @@
       end
     end
   end
+
+  path "/api/reactions" do
+    describe "post to create reaction" do
+      post("create reaction") do
+        tags "reactions"
+        description(<<-DESCRIBE.strip)
+        This endpoint allows the client to create a reaction to a specified reactable (eg, Article, Comment, or User). For examples:
+        * "Like"ing an Article will create a new "like" Reaction from the user for that Articles
```

Comment:
Does the newly created 'Like' get returned in this case as well?

Reply:
Yes, same response format:

```
200 OK
{
    "result": "create",
    "category": "like",
    "id": 160,
    "reactable_id": 10,
    "reactable_type": "Article"
}
```

On repeat post, you'd see:

```
200 OK
{
    "result": "none",
    "category": "like",
    "id": 161,
    "reactable_id": 10,
    "reactable_type": "Article"
}
```

Looking at that now, I'm thinking the first response ought to be `201`.